sessions.md

```diff
@@ -0,0 +1,210 @@
+<a name="#top">Back to Top</a>
+
+# Sessions
+
+
+## Table of Contents
+
+* [Options](#Options)
+  * [httpOnly](#http-only)
+  * [secure](#secure)
+  * [path](#path)
+  * [domain](#domain)
+  * [validationStrategy](#validation-strategy)
+* [Incoming Request Handling](#incoming-request-handling)
+* [/me Handling](#me-handling)
+
+
+## Feature Description
+
+This document describes the how our framework integrations should handle user
+sessions and session management.
+
+There are a few things we'll do in respect to user sessions:
+
+- We'll always store an access and refresh token in the browser,
+these tokens will be obtained by using the password grant flow on
+the configured Stormpath application
+
+- This cookie will always be set with the `HttpOnly` flag set, meaning that no
+  Javascript access will be allowed to this cookie.
+
+- This cookie will be set with the `secure` flag if the application is in
```

Comment:
What determines 'production'?  

Currently the Java SDK will allow these if the request is being served by and being requested by localhost, for development ease-of-use.   For the purpose of this discussion, I'll call this 'development'.  

Is there any difference between 'production' and not 'development' as I've defined it?